src/viewer.js

```diff
@@ -1371,6 +1371,68 @@ class VolumeImageViewer {
       })
     })
 
+    let clickEvent = null
+    this[_map].on('dblclick', (event) => {
+      clickEvent = 'dblclick'
+      this[_map].forEachFeatureAtPixel(
+        event.pixel,
+        (feature) => {
+          const correctFeature = feature.values_ && feature.values_.features ? feature.values_.features[0] : feature
+          console.debug('dblclick feature id:', correctFeature)
+          if (correctFeature && correctFeature.getId()) {
```

Comment:
Instead of
`feature.values_ && feature.values_.features ?` and `correctFeature && correctFeature.getId()`

you can make things easier to understand by using the optional chaining operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

So those lines would become:

`feature?.values_?.features ?` and `correctFeature?.getId()`

Which are equivalent to the old ones and much simpler.